src/pages/home/ProfileSetup.jsx

```diff
@@ -46,6 +42,9 @@ export default function ProfileSetup({ userData }) {
       setNoNameError(true);
     }
   }
+
+  useOnEnter(saveProfile);
```

Comment:
I'm not clear on what this does.

Reply:
`useOnEnter` is a new hook I made, basically when the user hits the enter key the `saveProfile` function will be called (submitting the form)